x-pack/plugins/apm/public/components/shared/charts/helper/get_chart_anomaly_timeseries.tsx

```diff
@@ -36,21 +44,22 @@ export function getChartAnomalyTimeseries({
 
   const boundaries = [
     {
-      title: 'model plot',
+      title: expectedBoundsTitle,
       type: 'area',
-      fit: Fit.Lookahead,
-      hideLegend: true,
+      fit: Fit.Nearest,
+      hideLegend: false,
       hideTooltipValue: true,
       areaSeriesStyle: {
         point: {
           opacity: 0,
         },
       },
-      color: rgba(theme.eui.euiColorVis1, 0.5),
+      color: anomalyTimeseriesColor ?? rgba(theme.eui.euiColorVis1, 0.5),
       stackAccessors: ['x'],
-      yAccessors: ['y0'],
-      y0Accessors: ['y1'],
+      yAccessors: ['y1'],
+      y0Accessors: ['y0'],
```

Comment:
As noted by @sqren in https://github.com/elastic/kibana/pull/132456#issuecomment-1131650796, the lower bound is always being plotted as `0` so the `y0` values for `model_lower` are never getting used. Is there an extra property that needs to be set on the chart to prevent the area always extending to `0`. The `anomalyTimeseries` passed to the chart here looks ok.